digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -824,7 +846,11 @@ Returns empty result as Json message, not as page not found
     params.get("bbox").map { bbox =>
       val boundingRectangle = constructBoundingRectangle(bbox)
       validateBoundingBox(boundingRectangle)
-      speedLimitService.withRoadAddress(speedLimitService.get(boundingRectangle, municipalities)).map { linkPartition =>
+      val speedlimits = params("withRoadAddress") match {

```

Comment:
Maybe you could repalce params("withRoadAddress") by withRoadAddress

Reply:
If you do the modification to Boolean type like I say above, in this case you can use:

if(params("withRoadAddress").toBoolean) {
..........
}